### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by Holybasil**

### CODE REVIEW

File: src/app/challenges/[slug]/ChallengeSubmit/CodeEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+import editorTheme from "@/theme/editorTheme";
+import Button from "@/components/Button";
+import { useParams } from "next/navigation";
+import SuccessAlert from "./SuccessAlert";
+
+enum TestStatus {
+  SUCCESS = "success",
+  ERROR = "error",
+  UNDEFIEND = "",
```
Comment (isabellewei) [inline src/app/challenges/[slug]/ChallengeSubmit/CodeEditor.tsx:16]: couple instances of this in the file, nit: 
```suggestion
  UNDEFINED= "",
```

File: src/app/challenges/submission/page.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import TallyForm from "@/components/TallyForm";
 const SubmissionPage = () => {
   return (
     <TallyForm
-      tallyUrl="https://tally.so/r/wLYDW1"
+      tallyUrl="https://tally.so/r/mRBp4d"
```
Comment (isabellewei) [inline src/app/challenges/submission/page.tsx:9]: I think this is the wrong URL? I'm getting the `Register for the Scroll Articles Bounty Contest: Write and Earn!` form

**Review by isabellewei — APPROVED** [review]
This looks so good! I'm excited for it to go live 🙌

File: src/app/challenges/submission/page.tsx (hunk 2 above)
Comment (Holybasil) [reply src/app/challenges/submission/page.tsx:9]: CleanShot 2024-11-12 at 22 06 51@2x
could you re-confirm it with Emily? @isabellewei

File: src/challenges/defi-challenges/simple-token-swap.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+level: 2
+website: "https://github.com/LevelUpWeb3/SimpleTokenSwap-Challenge"
+summary: Introduction to implementing a simple token swap contract by understanding Uniswap swap function.
+heroImage: "/images/level-up-preview.png"
+labels: ["DeFi"]
+---
+
+# Simple Token Swap
+
+If you haven't completed the ERC20 basic challenge yet, check it out [here](https://www.levelup.xyz/challenges/ERC20)!
```
[the remark this answers is not in the document]
Comment (Holybasil) [reply src/challenges/defi-challenges/simple-token-swap.mdx:13]: pick from https://github.com/LevelUpWeb3/app/pull/57/files#diff-86319f4014a641706beb5ba95b97bb965ac858be7c7d201b98caad0ea9317fd4, could you confirm with Emily?

[the remark this answers is not in the document]
Comment (isabellewei) [reply src/challenges/defi-challenges/simple-token-swap.mdx:13]: fixed

**Review by isabellewei — APPROVED** [review]
LGTM!

**Pull request merged**